shuffle/cli/src/test.rs

```diff
@@ -138,26 +139,25 @@ fn host_and_port(url: &Url) -> Result<String> {
     ))
 }
 
-pub fn run_move_unit_tests(project_path: &Path) -> Result<()> {
-    let unit_test_cmd = cli::PackageCommand::UnitTest {
-        // Setting to default values with exception of report_storage_on_error
-        // to get more information about a failed test
+pub fn run_move_unit_tests(project_path: &Path) -> Result<UnitTestResult> {
+    let unit_test_config = UnitTestingConfig {
         instruction_execution_bound: 5000,
         filter: None,
         list: false,
         num_threads: 8,
         report_statistics: false,
         report_storage_on_error: true,
         check_stackless_vm: false,
-        verbose_mode: false,
-        compute_coverage: false,
+        verbose: false,
+        ..UnitTestingConfig::default_with_bound(None)
```

Comment:
Nit: I think you can probably remove most of these fields now that you are using the `UnitTestingConfig::default_with_bound`. I think the only one that would need to stay is `report_storage_on_error`